lib/folio_client.rb

```diff
@@ -82,8 +82,8 @@ def configure(url:, login_params:, okapi_headers:, timeout: default_timeout, leg
              :fetch_marc_hash, :fetch_marc_xml, :get, :has_instance_status?,
              :http_get_headers, :http_post_and_put_headers, :interface_details,
              :job_profiles, :organization_interfaces, :organizations, :users, :user_details,
-             :post, :put, to:
-      :instance end
+             :post, :put, to: :instance
+  end
```

Comment:
🎉

Reply:
My own editor did that in a prior commit... 😩 😆